README.rst

```diff
@@ -40,3 +40,10 @@ Future work
 * Support moving messages
 * Support multiple mailboxes
 * Support alternate text-to-speech providers
+
+
+Problems
+============
+On Home Assistant there might be a problem reading the CDR file because of missing time.
```

Comment:
This doesn't belong here.  File a pull request on home-assistant instead.  Or I can do it if you don't want to

Reply:
I thought that it would help others until the problem would be resolved.